Flow.Launcher/ViewModel/ResultViewModel.cs

```diff
@@ -84,6 +84,19 @@ public Visibility ShowIcon
             }
         }
 
+        public double IconRadius
+        {
+            get
+            {
+                if (Result.RoundedIcon)
```

Comment:
I feel like this is a bit hacky🤣
~~And also you may need a OnPropertyChanged?~~ (maybe not because fody handles that)